wcivf/apps/hustings/management/commands/import_hustings.py

```diff
@@ -119,22 +143,14 @@ def handle(self, **options):
         if options["quiet"]:
             self.stdout = open(os.devnull, "w")
 
-        for_election = options["date"]
-
-        self.delete_all_hustings(for_election)
-        hustings_counter = 0
-        self.not_a_constituency_friend = []
-        with open(options["filename"], "r") as fh:
-            reader = csv.DictReader(fh)
-            for row in reader:
-                try:
-                    husting = self.create_husting(row)
-                except:
-                    husting = None
-                if husting:
-                    hustings_counter += 1
-                    self.stdout.write(
-                        "Created husting {0} <{1}>".format(
-                            hustings_counter, husting
-                        )
-                    )
+        count, _ = Husting.objects.all().delete()
```

Comment:
I think it's good to do this, but this wouldn't work well if we tried to use the old method of throwing a file at this command - it would remove all hustings, add in the ones from the file, but then delete them again next time it was run with no arguments.

This might be really useful for local development, but I think it should come with a warning

Reply:
Yeah this is true - it was one of the complications of trying to keep it backwards compatible. But I wasn't really sure when we would want to import from a physical file other than local dev now (I might be wrong). I've added a user prompt to confirm when using a file, do you think that is reasonable?